src/lib/features/personal-dashboard/personal-dashboard-service.ts

```diff
@@ -173,6 +183,11 @@ export class PersonalDashboardService {
             insights: {
                 avgHealthCurrentWindow,
                 avgHealthPastWindow,
+                totalFlags,
+                potentiallyStaleFlags,
+                staleFlags,
+                activeFlags: totalFlags - staleFlags - potentiallyStaleFlags,
```

Comment:
Don't we also get the total flags in the payload? Any reason we calculate again?

Reply:
We do get it in the personal dashboard global overview. This is the project details endpoint. Theoretically we could reuse the value from the global overview but this was just convenient. If we ever decide to re-use the project endpoint we get all the data in one go.

Reply:
So in other words this change allows to move the project details around without thinking about the data source since it's self sufficient.